charts/atlas-cluster/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: atlas-cluster
+description: A Helm chart to manage Atlas resources with Atlas operator
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.0.0"
```

Comment:
shouldn't this be equal to AO release to specify the version of the CRDs that is supported by the current chart?

Reply:
yep. will get it fixed